discussion.md

```diff
@@ -6,4 +6,31 @@ subtitle: Discussion
 
 ## Frequently Asked Questions
 
-?
+### Why doesn't Travis-CI run my tests?
+
+A common problem in the [Continuous Integration](./08-ci.html) part of this 
+lesson is that Travis-CI does not run the tests in the version control
```

Comment:
I usually write this as "version-controlled repository" rather than "version control repository"